couler/core/step_update_utils.py

```diff
@@ -221,14 +221,20 @@ def _get_params_and_artifacts_from_args(args, input_param_name, prefix):
                 i += 1
         else:
             if isinstance(values, OutputArtifact):
-                artifacts.append(
-                    {
-                        "name": utils.input_parameter_name(
-                            input_param_name, i
-                        ),
-                        "from": values,
-                    }
-                )
+                tmp = values.value.split(".")
+                if len(tmp) < 5:
```

Comment:
What's the magic 5 here? Perhaps either add some comments or converting to named fields using named tuple, etc.

Reply:
@javoweb is working on making this a function similar to [this one](https://github.com/couler-proj/couler/blob/254844533002eb9404746c0ccd01c0e712e5f413/couler/core/templates/output.py#L51).